src/duqtools/ids/_handle.py

```diff
@@ -131,11 +131,12 @@ def is_local_db(self):
 
     def path(self, suffix=SUFFIXES[0]) -> Path:
         """Return location as Path."""
+        imas_home = os.environ.get('IMAS_HOME')
+
         if self.is_local_db:
             template = LOCAL_PATH_TEMPLATE
-        elif self.user == 'public':
-            template = str(os.environ['IMAS_HOME'] + '/' +
-                           PUBLIC_PATH_TEMPLATE)
+        elif imas_home and self.user == 'public':
```

Comment:
it seems a bit tricky to get the path like this, as IMAS might interpret the path differently, dependent on the backend. I understand the `if imas_home` here for testing. But it is not the same path that the IMAS MDS backend would compute if it wouldn't exist (it would just crash). 

I looked into getting the path from IMAS itself, but that does not seem to be available. This is just so that we are aware of this, no action needed

Reply:
Yeah, I understood that. I added the if statement, because if `IMAS_HOME` is not defined, this crashes duqtools whenever the path is requested.